iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
+      titleLabel?.attributedText = nil
+      return
+    }
+
+    let titleAttributes: [NSAttributedString.Key: Any] = [
+      .font: titleFont,
+      .foregroundColor: UIColor.blackPrimaryText()
+    ]
+
+    let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+
+    guard let secondaryTitle = secondaryTitle,
+          let secondaryTitleFont = StyleManager.shared.theme?.fonts.medium16 else {
+      titleLabel?.attributedText = attributedText
+      return
+    }
+
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.paragraphSpacingBefore = 8
+    let secondaryTitleAttributes: [NSAttributedString.Key: Any] = [
+      .font: secondaryTitleFont,
+      .foregroundColor: UIColor.blackPrimaryText(),
+      .paragraphStyle: paragraphStyle
+    ]
+
+    attributedText.append(NSAttributedString(string: "\n" + secondaryTitle, attributes: secondaryTitleAttributes))
```

Comment:
You likely don't need to attribute "\n". But it may be ok for simplicity.

Reply:
`"\n"` manages layout here. And `paragraphStyle` with `paragraphSpacingBefore` starts working.

Reply:
The question is: does `paragraphSpacingBefore` apply also to the first title line with this approach? If no, then it's ok.

Reply:
No, is't applied to the second line only.